src/server.h

```diff
@@ -1308,6 +1319,10 @@ struct redisServer {
     int lfu_log_factor;             /* LFU logarithmic counter factor. */
     int lfu_decay_time;             /* LFU counter decay factor. */
     long long proto_max_bulk_len;   /* Protocol bulk length maximum size. */
+    /* PMEM */
+    int memory_alloc_policy;          /* Policy for memory allocation */
+    unsigned int pmem_threshold;      /* Persistent Memory threshold. */
+    ratioDramPmemConfig pmem_ratio;   /* Persistent Memory ratio. */
```

Comment:
I suggest dram_pmem_ratio or something similar instead of "pmem_ratio" which refers only to PMEM.